pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +214,23 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("bindings", "endpoints", "persistentvolumes", "persistentvolumes/status",
```

Comment:
@liggitt  - I trimmed down storage admin based on what you mentioned, but I still left the ability for the storage-admin to create endpoints and services based on the idea if I'm a gluster storage-admin, and I'm setting up SC's and PV's for access to my storage, wouldn't I also want to be able to setup the endpoints for gluster as well without having to add an additional step of the cluster-admin giving me access to a particular project to do so?  thoughts?  Also, I left some read perms in for some stuff that was thinking might be useful for a storage-admin to be able to view.